Digipost/APIClient.swift

```diff
@@ -360,26 +339,26 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                 self.uploadProgress?.completedUnitCount = totalSent
             })
         }
+        validateFullScope { () -> Void in
+            let task = self.fileTransferSessionManager.dataTaskWithRequest(urlRequest, completionHandler: { (response, anyObject, error) -> Void in
+                dispatch_async(dispatch_get_main_queue(), {
+                    self.removeTemporaryUploadFiles()
+                    self.isUploadingFile = false
+                    if self.isUnauthorized(response as! NSHTTPURLResponse?) {
```

Comment:
Probably not, the error will be handled by the other error hander. I can remove it!